README.md

```diff
@@ -14,22 +14,26 @@ VPF works on Windows and Linux. The requirements are as follows
 
 ### Linux
 
-We recommend Ubuntu 22.04 as it comes with a recent enough ffmpeg system packages.
+We recommend Ubuntu 22.04 as it comes with a recent enough FFmpeg system packages.
+For older versions of Ubuntu, we recommend building and installing FFmpeg from source:
+https://docs.nvidia.com/video-technologies/video-codec-sdk/12.0/ffmpeg-with-nvidia-gpu/index.html
 ```bash
-# Install dependencies (replace XXX in libnvidia-encode-XXX, libnvidia-decode-XXX with your driver version)
-# libnvidia-encode is part of the driver meta package so if your driver works fine you might already have it ! 
+# Install dependencies
 apt install -y \
           libavfilter-dev \
           libavformat-dev \
           libavcodec-dev \
           libswresample-dev \
           libavutil-dev\
           wget \
-          cmake \
           build-essential \
-          git \
-          libnvidia-encode-XXX \
-          libnvidia-decode-XXX 
+          git
+
+# Check if Nvidia video driver is installed! On Ubuntu this can be done by installing the following packages:
+apt install -y libnvidia-encode-XXX  libnvidia-decode-XXX 
+# Replace libnvidia-encode-XXX, libnvidia-decode-XXX with your driver version!
+# Important: They need to be installed on the host when using Docker containers!
```

Comment:
I would prefer just using the meta package here 🤔 Or even state that if you have a working driver installed they are installed as well ?

Reply:
Are you sure I think this is how i got them 🤔

Reply:
at least modern driver meta packages include both.